pkgs/by-name/xp/xpra-html5/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenv,
```

```suggestion
  stdenvNoCC,
```
This probably is enough.